### DISCUSSION TITLE
Any way to change the default volume changer?

### DISCUSSION THREAD

**Discussion by x8c8r** [HaxeFlixel/flixel#2317] [Q&A]
You can change the preloader, you can disable autopause. But, can you change or disable the default, pixel volume changer on "+" and "-" keys?

**Comment by Gama11** [comment]
Yes, actually, you can subclass `FlxGame()` and set `_customSoundTray` to a custom subclass of [`FlxSoundTray`](https://github.com/HaxeFlixel/flixel/blob/bfe851236c234db8e693990fc155f94b93682b29/flixel/system/ui/FlxSoundTray.hx):

https://github.com/HaxeFlixel/flixel/blob/bfe851236c234db8e693990fc155f94b93682b29/flixel/FlxGame.hx#L173-L177

To disable the sound tray entirely you can define `FLX_NO_SOUND_TRAY`.

**Reply by x8c8r** [reply]
Thank you, but how do I make custom sound tray?

**Reply by x8c8r** [reply]
Actually, lemme think about it myself. Thank you once again

**Reply by x8c8r** [reply]
Nah, I have no idea what to do

**Reply by x8c8r** [reply]
Okay, so I basically copypasted the file to my game sources. Renamed it to `flixel.system.ui.FlxSoundTray.hx`. And now I wonder, how do I include it in my Main.hx

**Reply by RichardBray** [reply]
I think maybe the idea was to create a new class that extends FlxGame, change your new FlxGame class then add that to the Main.hx of your game.

```hx
class CustomFlxGame extends FlxGame {
	public function new() {
		super(1920, 1080, startLevel);
		_customSoundTray = "your sound tray";	
	}
}
class Main extends Sprite {
	public function new() {
		super();
		addChild(new CustomFlxGame());
	}
}
```

I haven't tried it but I think this is what it should look like. Let me know if that works 🤞

**Reply by x8c8r** [reply]
I can't really try it now, but I will tomorrow. Thank you!
